README.md

```diff
@@ -1,60 +1,49 @@
-# generator-react-zeal
-
-Yeoman generator for Zeal's React boilerplate
-
-Expected to be used in the root of a Rails or Phoenix project configured with
-the Zeal client app strategy.
-
+### Yeoman Generator for [Zeal's](https://codingzeal.com/) React Boilerplate
```

Comment:
I think the general pattern for GitHub repos is to have the name of the project as an h1 at the top of the README, so I think we should keep the title and tag line as they were.

But 👍 on adding the link to our website!  And 👍 on removing the Rails/Phoenix paragraph